Rakefile

```diff
@@ -78,3 +78,19 @@ task :benchmark, [:name] do |_t, args|
     end
   end
 end
+
+# Run profiling on a benchmark and generate a flamegraph.
+# To run this task, you need to have stackprof installed.
+# https://github.com/tmm1/stackprof
+# To run profiling for a specific benchmark you can run:
+#   bundle exec rake prof\[bench_slow_query\]
+desc 'Profile a benchmark (e.g., profile[bench_slow_query])'
```

Comment:
@volodya-lombrozo maybe we should use this target in CI? Otherwise, how do we know it works?